src/components/MissingDataPopup/MissingDataPopup.module.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../styles/variables';
+
+.popupWrapper {
+    cursor: pointer;
+    display:inline-block;
+    padding-right: 3.75em;
+}
+
+.popupChoice {
+    display:inline-block;
+    padding:5px;
+    margin:5px 10px 5px 0px;
+    min-width: 3em;
+    cursor:pointer;
+    background: $black;
+    text-align:center;
+    border-radius: 15%;
+    font-weight: 800;
+
+    &:hover{
+        background: #222;
+    }
+
+    &.selected{
+        background: rgb(94, 138, 160);
```

Comment:
Is this a color which may be used elsewhere? We might want to define it as a variable in styles